src/pkg/utils/utils.go

```diff
@@ -70,3 +71,12 @@ func StringSliceContains(arr []string, elem string) bool {
 	}
 	return false
 }
+
+// CheckIfRemoteURL returns "true" if filePath is a remote file
+func CheckIfRemoteURL(filePath string) bool {
```

Comment:
nit: I think this function is pretty specific to the anthos installer behavior. IMO we should put the function in install_packages_step.go privately for now. If we find another need for it, we can put it in a more accessible package and export it.